src/main/java/pl/javastart/task/Number.java

```diff
@@ -0,0 +1,38 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class Number {
+    private int number;
```

Comment:
Staraj się ograniczać zasięg zmiennych, czyli jeżeli coś nie musi być polem klasy, to preferuj zmienne lokalne i parametry metod